device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/Message.java

```diff
@@ -35,6 +35,16 @@
      */
     private String messageId;
 
+    /**
+     * Whether or not there is a list of nested messages included in this instance.
+     */
+    private boolean isBulk;
+
+    /**
+     * List of nested messages.
+     */
+    private Set<Message> nestedMessages;
+
```

Comment:
so you can use lombok getter here right instead of line 194?

Reply:
True, but then I will have to change all the getters in this class no?